src/hotspot/share/opto/loopnode.hpp

```diff
@@ -775,6 +771,8 @@ class IdealLoopTree : public ResourceObj {
 
   void remove_main_post_loops(CountedLoopNode *cl, PhaseIdealLoop *phase);
 
+  void collect_loop_core_nodes(PhaseIdealLoop* phase, Unique_Node_List& wq) const;
+
```

Comment:
nit: why move it?

Reply:
This function was private before. I need to make it public so I can use it in `vmaskloop.cpp`.